MH4U Dex/ItemEncyclopediaTableViewController.m

```diff
@@ -46,12 +42,12 @@ - (void)viewDidLoad
 
 - (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView
 {
-    return 1;
+    return self.fetchedResultsController.sections.count;
 }
 
 - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
 {
-    return self.items.count;
+    return [[self.fetchedResultsController.sections objectAtIndex:section] numberOfObjects];
```

Comment:
Actually, `numberOfObjects` is a property, too, so this should work (but Xcode might complain because it doesn't know where `numberOfObjects` is supposed to come from):

```objc
return self.fetchedResultsController.sections[section].numberOfObjects;
```